index.html

```diff
@@ -24,6 +24,214 @@ <h2 class="support">I’am a Software Developer</h2>
       <li><a href="https://twitter.com/GhilainIshimwe"><img src="image/twitter.png" alt="twitter logo"></a></li>
       <li><a href="https://www.microverse.org/"><img src="image/gmail.png" alt="social logo"></a></li>
     </ul><img src="image/illustration-header-mobile.png" class="illustrate" alt="vector image">
+  </section>
+  <div class="recent-work">
+    <div class="frame44">
+      <div class="frame43">
+        <div class="cardwork1">
+          <div class="container">
+            <div class="action">
+            <button class="buton1" type="button"> See Project</button>
+          </div>
+         <div class="supporttext">
+          <p class="supporttext1">A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+            </div>
+            <div class="titlpost">Profesional Art Printing Data
+          </div>
+          <div class="tag">
+            <div class="ruby"> Ruby</div>
+            <div class="bootstrap"> bootstrap</div>
+            <div class="html">html</div>
+          </div>
```

Comment:
- [ ] Kindly use `ul` instead of `div` here `<div class="tag">` and `li` instead of `div` for each technology. Apply the same for the rest of the projects.